apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -365,7 +365,7 @@ function EventTypeSingleLayout({
           <Button
             className="ml-4 lg:ml-0"
             type="submit"
-            loading={formMethods.formState.isSubmitting || isUpdateMutationLoading}
```

Comment:
This `isSubmitting` boolean never goes to false when an error throws. That resulted in the user needing to refresh if an error occurs. Using only the mutation boolean seems sufficient here.